src/commands/install.rs

```diff
@@ -1,113 +1,129 @@
-use std::fmt::Write;
-use std::fs;
-
 use anyhow::{bail, Result};
 use clap::Parser;
+use std::fmt::Write;
+use std::fs;
 use tracing::{error, info};
 
-use crate::constants::{FUEL_CORE_RELEASE_DOWNLOAD_URL, SWAY_RELEASE_DOWNLOAD_URL};
-use crate::download::{download_file_and_unpack, fuelup_bin_dir, unpack_extracted_bins};
-use crate::{
-    constants::{FUEL_CORE_REPO, GITHUB_API_REPOS_BASE_URL, RELEASES_LATEST, SWAY_REPO},
-    download::{forc_bin_tarball_name, fuel_core_bin_tarball_name, get_latest_tag},
+use crate::download::{
+    component, download_file_and_unpack, fuelup_bin_dir, unpack_extracted_bins, DownloadCfg,
 };
 
 #[derive(Debug, Parser)]
-pub struct InstallCommand {}
+#[clap(override_usage = "\
+    fuelup install <COMPONENT>[@<VERSION>] ...")]
```

Comment:
Woops I think we want to remove this override too?

Reply:
Thanks for the catch!